### DISCUSSION TITLE
Gradle build failed: Unresolved reference: inDir

### DISCUSSION THREAD

**Discussion by KalleChen** [daniel-seiler/ImageTheming#1] [Q&A] [answered]
<img width="738" alt="Screen Shot 2021-11-17 at 9 48 04 AM" src="https://user-images.githubusercontent.com/48475056/142094229-122e3ba8-3198-4427-aeeb-3fcc6a29e4ea.png">

Just get this error while using `gradle jar` to build a jar file. I'm not familiar with gradle and kotlin. I have searched for some similar errors and found that it might be the problem with the version. But I have no idea. Is there something I went wrong?

## Environment 
I use https://hub.docker.com/_/gradle gradle's official docker image to build the gradle enviornment.
host: macOS Monterey 12.0.1 MacBook Air (M1 2020)

**Comment by daniel-seiler** [comment, marked as answer]
Thanks for bringing this to my attention. Your error actually refers to an incorrect function call (Unresolved reference: inDir). Besides that, the jar file generated by `gradle jar` still wouldn't work because of missing dependencies. These issues are fixed now

**Reply by KalleChen** [reply]
Got it, thanks for your reply and fixing.😀
